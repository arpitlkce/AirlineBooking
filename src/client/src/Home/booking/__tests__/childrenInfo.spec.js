import React from 'react';
import renderer from 'react-test-renderer';
import ChildrenInfo from '../childrenInfo';

describe('ChildrenInfo Component', () => {
    it('should render correctly in  mode', () => {
        const component = renderer.create(<ChildrenInfo />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
