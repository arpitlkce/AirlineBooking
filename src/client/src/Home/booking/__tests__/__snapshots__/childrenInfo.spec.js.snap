// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChildrenInfo Component should render correctly in  mode 1`] = `
<div
  className="paddingBottom container"
>
  <div
    className="marginInfo row"
  >
    <div
      className="col-md-6"
    >
      <input
        className="form-control"
        name="firstName"
        onChange={[Function]}
        placeholder="First Name"
        type="text"
      />
    </div>
    <div
      className="col-md-6"
    >
      <input
        className="form-control"
        name="lastName"
        onChange={[Function]}
        placeholder="Last Name"
        type="text"
      />
    </div>
  </div>
  <div
    className="marginInfo row"
  >
    <div
      className="col-md-6"
    >
      <div
        className="btn-group"
        role="group"
      >
        <button
          aria-label={null}
          className="btn btn-outline-primary"
          name="gender"
          onClick={[Function]}
          type="button"
          value="male"
        >
          Male
        </button>
        <button
          aria-label={null}
          className="btn btn-outline-primary"
          name="gender"
          onClick={[Function]}
          type="button"
          value="female"
        >
          Female
        </button>
      </div>
    </div>
    <div
      className="col-md-6"
    >
      <input
        className="form-control"
        name="age"
        onChange={[Function]}
        placeholder="Age"
        type="text"
      />
    </div>
  </div>
</div>
`;
